### DISCUSSION TITLE
superagent-api doesn't start in docker. setup.docker.sh is missing

### DISCUSSION THREAD

**Discussion by ppokrovskii** [superagent-ai/superagent#435] [Q&A] [answered]
superagent-api doesn't start in docker. error from logs:
> /bin/bash: setup.docker.sh: No such file or directory

I guess I should just copy and rename replit.sh into setup.docker.sh?
or just remove `command: ["/bin/bash", "setup.docker.sh"]` from _docker-compose.yml_ as far as `RUN prisma generate` is executed as part of dockerfile?

**Comment by homanp** [comment, marked as answer]
> superagent-api doesn't start in docker. error from logs:
> 
> > /bin/bash: setup.docker.sh: No such file or directory
> 
> 
> 
> I guess I should just copy and rename replit.sh into setup.docker.sh?
> 
> or just remove `command: ["/bin/bash", "setup.docker.sh"]` from _docker-compose.yml_ as far as `RUN prisma generate` is executed as part of dockerfile?
> 
> 

Yes! @bbfred any ideas here?
